src/usdb_syncer/settings.py

```diff
@@ -137,6 +140,60 @@ def default() -> Newline:
         return Newline.LF
 
 
+class FixLinebreaks(Enum):
+    """Supported variants for fixing linebreak timings."""
+
+    DISABLE = 0
+    USDX_STYLE = 1
+    YASS_STYLE = 2
+
+    def __str__(self) -> str:
+        match self:
+            case FixLinebreaks.DISABLE:
+                return "disable"
+            case FixLinebreaks.USDX_STYLE:
+                return "USDX style"
+            case FixLinebreaks.YASS_STYLE:
+                return "YASS style"
+            case _ as unreachable:
+                assert_never(unreachable)
+
+
+class FixFirstWordsCapitalization(Enum):
```

Comment:
As it's a simple switch, a bool (i.e. a checkbox in the GUI) would be sufficient.